hedgehog/hedgehog.cabal

```diff
@@ -112,6 +112,11 @@ library
     Hedgehog.Internal.Tree
     Hedgehog.Internal.Tripping
 
+  other-modules:
+    -- This has nothing to do with Hedgehog proper, and it's
+    -- too trivial to bother exposing.
+    Hedgehog.Internal.Predicate
```

Comment:
I agree with the comment, but would like to maintain a policy of always exposing all internal modules

Reply:
https://github.com/hedgehogqa/haskell-hedgehog/pull/339#discussion_r346418133, most likely we don't even need to introduce a new module.